content/blog/2019/06/2019-06-25-phase-1-multibranch-pipeline-support-for-gitlab.adoc

```diff
@@ -0,0 +1,324 @@
+---
+layout: post
+title: "Multi-branch Pipeline Jobs Support for GitLab SCM"
+tags:
+- gitlab
+- plugins
+- pipeline
+- credentials
+- developer
+- gsoc
+- gsoc2019
+author: baymac
+---
+
+= Multi-branch Pipeline Support for GitLab SCM
+
+This is one of the Jenkins project in GSoC 2019. We are working on adding support for link:/projects/gsoc/2019/gitlab-support-for-multibranch-pipeline/[Multi-branch Pipeline Jobs and Folder Organisation in GitLab]. The plan is to create the following
+plugins:
+
+1. link:https://github.com/jenkinsci/gitlab-api-plugin[GitLab API Plugin] - Wraps GitLab Java API
+
+2. link:https://github.com/baymac/gitlab-branch-source-plugin[GitLab Branch Source Plugin]:
```

Comment:
```suggestion
. link:https://github.com/baymac/gitlab-branch-source-plugin[GitLab Branch Source Plugin]:
```

Do this for all ordered lists in this document - asciidoc will autonumber for you.  

See https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#lists - particularly, "Complex content in outline lists" shows how to do continuations using `+` to have a list with empty lines inside items.